src/components/FeatureViews/Registration/RegistrationLanding.js

```diff
@@ -97,98 +107,118 @@ const RegistrationLanding = () => {
-		}, Object.values(courses));
-
-	const handleFilterChange = (filterType) => (filters) => {
-            setCourseFilters((prevFilters) => ({
-                ...prevFilters,
-                [filterType]: filters || [],
-            }));
-	};
-
-	const renderFilter = (filterType) => {
-            let options = [];
-            switch (filterType) {
-                case "instructor":
-                    options = instructorOptions;
-                    break;
-                case "subject":
-                    options = subjectOptions;
-                    break;
-                case "grade":
-                    options = gradeOptions;
-                    break;
-                // no default
+                    return courses;
             }
+        }, Object.values(courses));
+
+    const handleFilterChange = (filterType) => (filters) => {
+        setCourseFilters((prevFilters) => ({
+            ...prevFilters,
+            [filterType]: filters || [],
+        }));
+    };
 
-            return (
-                <SearchSelect
-                    className="filter-options"
-                    closeMenuOnSelect={false}
-                    components={{ClearIndicator}}
-                    isMulti
-                    onChange={handleFilterChange(filterType)}
-                    options={options}
-                    placeholder={`All ${filterType}s`}
-                    styles={customStyles}
-                    value={courseFilters[filterType]}
-                />
-            );
-	};
+    const renderFilter = (filterType) => {
+        let options = [];
+        switch (filterType) {
+            case "instructor":
+                options = instructorOptions;
+                break;
+            case "subject":
+                options = subjectOptions;
+                break;
+            case "grade":
+                options = gradeOptions;
+                break;
+            // no default
+        }
+
+        return (
+            <SearchSelect
+                className="filter-options"
+                closeMenuOnSelect={false}
+                components={{ ClearIndicator }}
+                isMulti
+                onChange={handleFilterChange(filterType)}
+                options={options}
+                placeholder={`All ${filterType}s`}
+                styles={customStyles}
+                value={courseFilters[filterType]}
+            />
+        );
+    };
 
     return (
         <Paper elevation={2} className="RegistrationLanding paper">
-            <BackButton/>
-            <hr/>
-            <RegistrationActions/>
+            <Grid container>
+                <Grid item md={1}>
+                    <BackButton />
+                </Grid>
+                <Grid item md={10}/>
+                <Grid item md={1}>
+                    <Button component={NavLinkNoDup}
+				    to="/registration/cart">
+                        <Badge badgeContent={renderCartItems()} showZero color="primary" >
```

Comment:
This could be simplified by:
```
<Badge badgeContent={numToCheckout === null ? 0 : numToCheckout} .../>
```
and we could remove the ```renderCartItems``` function.

Reply:
@jsflca2012 this still hasn't been addressed